Common/Orders/Fees/ExanteFeeModel.cs

```diff
@@ -0,0 +1,122 @@
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using QuantConnect.Securities;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Orders.Fees
+{
+    /// <summary>
+    /// Provides an implementation of <see cref="FeeModel"/> that models Exante order fees.
+    /// According to:
+    /// <list type="bullet">
+    ///   <item>https://support.exante.eu/hc/en-us/articles/115005873143-Fees-overview-exchange-imposed-fees?source=search</item>
+    ///   <item>https://exante.eu/markets/</item>
+    /// </list>
+    /// </summary>
+    public class ExanteFeeModel : FeeModel
+    {
+        private readonly decimal _forexCommissionRate;
+
+        /// <summary>
+        /// Creates a new instance
+        /// </summary>
+        /// <param name="forexCommissionRate">Commission rate for FX operations</param>
+        public ExanteFeeModel(decimal forexCommissionRate = 0.25m)
+        {
+            _forexCommissionRate = forexCommissionRate;
+        }
+
+        /// <summary>
+        /// Gets the order fee associated with the specified order.
+        /// </summary>
+        /// <param name="parameters">A <see cref="OrderFeeParameters"/> object
+        /// containing the security and order</param>
+        /// <returns>The cost of the order in a <see cref="CashAmount"/> instance</returns>
+        public override OrderFee GetOrderFee(OrderFeeParameters parameters)
+        {
+            var order = parameters.Order;
+            var security = parameters.Security;
+
+            decimal feeResult;
+            string feeCurrency;
+            switch (security.Type)
+            {
+                case SecurityType.Forex:
+                    var totalOrderValue = order.GetValue(security);
+                    feeResult = Math.Abs(_forexCommissionRate * totalOrderValue);
+                    feeCurrency = Currencies.USD;
+                    break;
+
+                case SecurityType.Equity:
+                    var equityFee = ComputeEquityFee(order);
+                    feeResult = equityFee.Amount;
+                    feeCurrency = equityFee.Currency;
+                    break;
+
+                case SecurityType.Option:
+                case SecurityType.IndexOption:
+                    var optionsFee = ComputeOptionFee(order);
+                    feeResult = optionsFee.Amount;
+                    feeCurrency = optionsFee.Currency;
+                    break;
+
+                case SecurityType.Future:
+                case SecurityType.FutureOption:
+                    feeResult = 1.5m;
```

Comment:
Wouldn't the fee depend on the contract quantity?

Reply:
All Exante can provide I described at:

> Exante's info on fees: https://support.exante.eu/hc/en-us/articles/115005873143-Fees-overview-exchange-imposed-fees?source=search. I think I covered all the parts. How do you propose to expand it?

Do you see any extra info on the subject? I don't.